### PR TITLE
Ec proofreading odata docs

### PR DESCRIPTION
**Pull request by ilagordhan**
Closed the previous pull request by mistake. Need to verify that change made by MS is incorporated.

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
If you close a PR, can always reopen it 
image

**Comment by ilagordhan** [comment]
Yes, I saw that when I pushed a further edit.
Ela

File: content/refguide/published-odata-resource.md [hunk 1]
```diff
@@ -1,68 +1,112 @@
...
 
 {{% /alert %}}
 
 ## 1 Adding or Editing a Resource
 
-The **Add resource** or **Edit** button in the Published OData Service document will open an editor window for the published resource. The editor window is separated in two parts: **Resource** and **Uniform Resource Identifier**. The former lets you specify which entity and entity attributes to expose, whereas the latter lets you customize the location where the resource will be published.
+### 1.1 Add a Resource
+
+Click **Add** in the **Resources** pane of the **Published OData Service** window to open the **Select Entity** window. Select an entity to publish and click **Select**.
+
```
Comment (Adam-Dupaski) [inline content/refguide/published-odata-resource.md]: Delete extra spaces between text and attachment - only need 1 space on either side of attachment.

File: content/refguide/published-odata-resource.md [hunk 2]
```diff
@@ -1,68 +1,112 @@
...
 
-![](attachments/16713722/16843930.png)
+![](attachments/published-odata-resource/Edit-published-resource.png)
 
-IBM DB2 does not support read-isolated data retrieval operations that are non-blocking in a multi-user environment. Therefore the data retrieved by OData might not be 100% consistent if the same data rows are modified concurrently by another user. For the details see this [IBM DB2](db2) page.
 
-## 2 Selecting Exposed Attributes & Associations
 
-When an entity to publish has been selected, press **Select** to open a dialog that allows you to select individual attributes to expose.
+It is possible to select another **Entity** or view the entity in the domain model by clicking **Show**. The [Exposed attributes and associations](#exatass) for the selected entity can be set in this window.
```
Comment (Adam-Dupaski) [inline content/refguide/published-odata-resource.md]: Capitalization of "Exposed" in the cross-reference is probably unnecessary - don't need to capitalize it like it is UI unless you are instructing the user to to click/select it (as you do in the next section).

File: content/refguide/published-odata-resource.md [hunk 3]
```diff
@@ -1,68 +1,112 @@
...
-
-**Note that enabling this does mean that retrieved data can be inconsistent**, because you're no longer retrieving data within a single transaction. For example, you are sorting on an Age attribute in an entity called Customer and retrieving customers with 1000 objects per page. Now a customer gets deleted in between two calls, it means that the customer with Age 23 at position 1001 now moves to position 1000, meaning that this object that you _would_ have gotten on the next page now moves to the first page and is not retrieved anymore. The other way around with data inserts in between calls can cause you to see duplicates. So only use this option when this kind of inconsistency is acceptable.
+The **Use paging** option is used to set a maximum number of objects per response, and include a link to the next set of objects. A client such as [Tableau](https://www.tableau.com) is able use this to display progress and automatically continue to follow the links until all the data is retrieved. The memory usage of the clients can be improved if paging is set to a reasonable page size.
```
Comment (Adam-Dupaski) [inline content/refguide/published-odata-resource.md]: Comma and singular verb "include" a little confusing here: does it mean "use to set and include", or "the option is used to set, and the option includes"? Either the comma should be removed or the verb should be changed.

**Comment by Adam-Dupaski** [comment]
@ilagordhan please fix the Travis build error for this PR: https://travis-ci.org/mendix/docs/builds/655908681?utm_source=github_status&utm_medium=notification

File: content/refguide/published-odata-resource.md (hunk 1 above)
Comment (ilagordhan) [reply content/refguide/published-odata-resource.md]: deleted

File: content/refguide/published-odata-resource.md (hunk 2 above)
Comment (ilagordhan) [reply content/refguide/published-odata-resource.md]: changed

File: content/refguide/published-odata-resource.md (hunk 3 above)
Comment (ilagordhan) [reply content/refguide/published-odata-resource.md]: Comma removed.

**Comment by ilagordhan** [comment]
I would appreciate some guidance on interpreting Travis build errors. Can we discuss next week?

**Comment by Adam-Dupaski** [comment]
Normally Travis build errors show up in the `Errors` section at the bottom of the log.

But the error here is in the doc metadata, so it shows up earlier in the log: `[07:39:53] [MENU BUILD] No metadata for file: /refguide/published-odata-resource.md, is the block at the top correct?`

You can check the metadata of the doc and find the mistake - it is a small one and easily fixable.

**Comment by ilagordhan** [comment]
Thanks for the heads up.

File: content/refguide/published-odata-resource.md [hunk 4]
```diff
@@ -3,66 +3,103 @@ title: "Published OData Resource"
...
 {{% /alert %}}
 
 ## 1 Adding or Editing a Resource
 
-The **Add resource** or **Edit** button in the Published OData Service document will open an editor window for the published resource. The editor window is separated in two parts: **Resource** and **Uniform Resource Identifier**. The former lets you specify which entity and entity attributes to expose, whereas the latter lets you customize the location where the resource will be published.
+### 1.1 Add a Resource
+
+Click **Add** in the **Resources** pane of the **Published OData Service** window to open the **Select Entity** window. Select an entity to publish and click **Select**.
+
+![Odata service page](attachments/published-odata-resource/published-odata-service.png)
```
Comment (Adam-Dupaski) [inline content/refguide/published-odata-resource.md]: "OData"

File: content/refguide/published-odata-resource.md [hunk 5]
```diff
@@ -3,66 +3,103 @@ title: "Published OData Resource"
...
+
+To add a resource, click the OData service name in the **Select Published Data Service** window, and click **Select**. 
+
+To create a new OData service and add the entity to it, click **New**  and enter the name of the service you want to create in the **Add Published OData Service** dialog box. 
+
+### 1.2 Edit a Resource
+
+In the **Resources** pane of the **Published OData Service** window, select a resource and click **Edit** to display the **Edit published resource** window. 
+
+![Edit published Odata diaglog box](attachments/published-odata-resource/Edit-published-resource.png)
```
Comment (Adam-Dupaski) [inline content/refguide/published-odata-resource.md]: "OData"
"diaglog" = typo

File: content/refguide/published-odata-resource.md (hunk 4 above)
Comment (ilagordhan) [reply content/refguide/published-odata-resource.md]: Thanks for the feedback. I was trying out using Visual Studio with the pull request extension and acting on the error notifications that it gives - no excuse - just focus was on the coding.

File: content/refguide/published-odata-resource.md (hunk 5 above)
Comment (ilagordhan) [reply content/refguide/published-odata-resource.md]: Ditto

**Comment by ilagordhan** [comment]
Thanks for the feedback.

**Pull request merged**